ups/display_ginga.cfg

```diff
@@ -1,4 +1,5 @@
 # -*- python -*-
+import lsst
```

Comment:
Other files like this (e.g., in afw) use `import lsst.sconsUtils`.

Reply:
OK.